Rubberduck.RibbonDispatcher/Rubberduck.RibbonDispatcher.sln.jnk

```diff
@@ -0,0 +1,36 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
I'm missing something. Why is there a solution file? Isn't Rubberduck.RibbonDispatcher assembly under Rubberduck.sln?

Reply:
I stuck the ".jnk" on the end to hide it, instead of deleting it outright. Caught with my hands in the cookie jar I guess.